packit/utils/__init__.py

```diff
@@ -59,3 +59,15 @@ def sanitize_branch_name_for_rpm(branch_name: str) -> str:
     for o in offenders:
         branch_name = branch_name.replace(o, "")
     return branch_name.replace("-", ".")
+
+
+def is_relative_to(original_path, *other):
+    """Copy of code from pathlib, because it is new feature and not in Fedora-32 and centos-8
+
+    Return True if the path is relative to another path or False.
+    """
+    try:
+        original_path.relative_to(*other)
+        return True
+    except ValueError:
+        return False
```

Comment:
I really had to check and this is actual copy-pasta of the [pathlib is_relative_to code](https://github.com/python/cpython/blob/1a08c5ac49b748d5e4e4b508d22d3804e3cd4dcc/Lib/pathlib.py#L812)